### PR TITLE
docs: refactor usage and playground

### PR DESCRIPTION
**Pull request by sh7dm**
## Description
Use new style for playgrounds

## Motivation and Context
This simplifies docs updates

## TODO:
- [x] Alerts
- [x] AppBars
- [x] Application
- [x] AspectRatios
- [x] Autocompletes
- [x] Avatars
- [x] Badges
- [x] Banners
- [x] BottomNavigation
- [x] BottomSheets
- [x] Breadcrumbs
- [x] ButtonGroups
- [x] Buttons
==================================== Yiping
- [ ] Calendars
- [x] Cards
- [x] Carousels
- [ ] Categories
- [x] ChipGroups
- [x] Chips
- [x] ColorPickers
- [x] Combobox
- [x] DataIterators
- [ ] DataTables
- [ ] DatePickers
- [ ] Dialogs
- [ ] Dividers
- [ ] ExpansionPanels
- [ ] FileInputs
- [ ] FloatingActionButtons
- [ ] Footer
- [ ] Forms
- [ ] GridLists
- [ ] Grids
========================================
- [x] Hover
- [x] Icons
- [x] Images
- [ ] Inputs
- [ ] ItemGroups
- [ ] Lazy
- [ ] ListItemGroups
- [ ] Lists
- [ ] Menus
- [ ] NavigationDrawers
- [ ] OverflowBtns
- [ ] Overlays
- [ ] Paginations
- [ ] Parallax
========================================
- [ ] ProgressCircular
- [ ] ProgressLinear
- [ ] Ratings
- [ ] SelectionControls
- [ ] Selects
- [ ] Sheets
- [ ] SimpleTables
- [ ] SkeletonLoaders
- [ ] SlideGroups
- [x] Sliders
- [ ] Snackbars
- [ ] Sparklines
========================================
- [ ] Steppers
- [ ] Subheaders
- [ ] SystemBars
- [ ] Tabs
- [ ] Textarea
- [ ] TextFields
- [ ] Timelines
- [ ] TimePickers
- [x] Toolbars
- [x] Tooltips
- [ ] Treeview
- [ ] Windows

## Types of changes

-  [ ] Bug fix (non-breaking change which fixes an issue)
-  [x] New feature (non-breaking change which adds functionality)
-  [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-  [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

-  [x] The PR title is no longer than 64 characters.
-  [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
-  [x] My code follows the code style of this project.
-  [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
-  [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/src/data/pages/components/FileInputs.json [hunk 1]
```diff
@@ -7,8 +7,14 @@
...
       "children": [
         {
-          "type": "usage",
-          "value": "usage"
+          "type": "usage-new",
+          "value": {
+            "booleans": [
+              "chips",
+              "counter",
+              "disabled"
```
Comment (sh7dm) [inline packages/docs/src/data/pages/components/FileInputs.json]:
```suggestion
"disabled",
"show-size"
```

Comment (YipingRuan) [reply packages/docs/src/data/pages/components/FileInputs.json]: image

Comment (sh7dm) [reply packages/docs/src/data/pages/components/FileInputs.json]: @YipingRuan thanks!

**Comment by YipingRuan** [comment]
@MajesticPotatoe  Should "update branch" button be hit?

Currently the branch CI fail without your fix on VDatePicker.month.spec.ts.
https://github.com/vuetifyjs/vuetify/commit/481f18f6a8658c50e938811650e1a721fc144de8

**Comment by YipingRuan** [comment]
@johnleider @MajesticPotatoe @sh7dm 
I suggest I do some boring plumbing work to make it easier.
Based on my observation, for each of the remaining components (XXX):

- Patch "packages/docs/src/data/pages/components/xxx.json".
````json
          {
          "type": "usage-new",
          "value": {
            "booleans": []
          }
````

-  Delete "packages/docs/src/examples/xxx/usage.vue"
-  Patch "packages/docs/src/lang/en/components/xxxxx.json". Move "examples.usage.desc" to "usageText"

- Add "packages/docs/src/usages/xxxx.vue"
````vue
<template>
</template>

<script>
  import Usage from './usage'
  export default {
    mixins: [Usage],
  }
</script>
````

Then you guys enrich based on that, make sense?

**Comment by johnleider** [comment]
@YipingRuan sounds good

**Pull request merged**